plugins/carbon_black_defense/komand_carbon_black_defense/connection/connection.py

```diff
@@ -94,16 +91,17 @@ def retrieve_results_for_detail_search(self, job_id: str):
         )
         return results
 
-    def call_api(self, method: str, url: str, params: dict = None, data: str = None, json_data: object = None) -> Optional[dict]:
+    def call_api(self, method: str, url: str, params: dict = None, data: str = None, json_data: object = None) -> json:
```

Comment:
This is a bit fishy to me. I think json objects are really just dictionaries, so I think this should be dict. Is json actually a type here?

Reply:
@pmara-r7 `json` appears as a type because Python is pretty loose with it's type hinting... 😅 @jrose-r7 the output type here should be `dict` if returning JSON.

Reply:
Updated output type to dict.